### ISSUE TITLE
Error: Cannot find module 'babel-minify-webpack-plugin'

### ISSUE THREAD

**Issue by dimitriharding** [gabrielcsapo/tap-html#7]
It seems like `babel-minify-webpack-plugin` should be moved from _devDependencies_ to _Dependencies_. 

I got the following error when I tried using this module. 

```console
> xo && nyc --reporter=html --reporter=text ava -s --tap | tap-html --out ./tap-html.html

module.js:491
    throw err;
    ^

Error: Cannot find module 'babel-minify-webpack-plugin'
    at Function.Module._resolveFilename (module.js:489:15)
    at Function.Module._load (module.js:439:25)
    at Module.require (module.js:517:17)
    at require (internal/module.js:11:18)
```

To resolve I had to do an npm install of `babel-minify-webpack-plugin`.

**Comment by gabrielcsapo** [comment]
Thanks for reporting this @dimitriharding fixing this now!

**Comment by gabrielcsapo** [comment]
This should be fixed in https://github.com/gabrielcsapo/tap-html/releases/tag/0.2.1. Please let me know if this works for you!

**Comment by dimitriharding** [comment]
Confirmed fix in `0.2.1`.

**Comment by gabrielcsapo** [comment]
thank you @dimitriharding!
